queries/perl/injections.scm

```diff
@@ -1 +1,2 @@
-(comments) @comment
```

Comment:
We globally have that comment injection; any reason to remove it?

Reply:
I'm not sure this is particularly a concsious decision to remove it, so much as just that our copy started off empty, and we added `(pod)`, and didn't add this one that you have.

That said, I don't think we have a `(comments)` definition, do we? Just a single-line `(comment)`.

Reply:
Well, whatever comment node you have should inject the `comment` parser.